fix: CLAUDE.md の markdownlint 違反（MD031）を修正

## 概要

全検証（pytest / ruff / mypy / markdownlint / shellcheck）で唯一残っている既存エラーを修正する。

## 検出された問題

### markdownlint MD031 (CLAUDE.md:68)

```
CLAUDE.md:68 error MD031/blanks-around-fences
Fenced code blocks should be surrounded by blank lines [Context: "```"]
```

コードブロック（` ``` `）の前後に空行がない。

## 全検証結果（2026-02-13 時点）

| ツール | 結果 |
|--------|------|
| pytest | 573 passed, 4 skipped（問題なし） |
| ruff | All checks passed |
| mypy | no issues found |
| markdownlint | **1 error**（本Issue対象） |
| shellcheck | SC1091 (info) x10 のみ（許容済み） |

## 対象ファイル

- `CLAUDE.md`

--- a/CLAUDE.md
+++ b/CLAUDE.md
@@ -66,6 +66,7 @@
     # Bad: ディレクティブ行にインラインコメント（パースエラー）
     # shellcheck disable=SC2016 -- reason here
     ```
+
 - ドキュメント内の図表（フローチャート、シーケンス図、ER図等）はmermaid形式を使用する
   - ASCII図表は使用しない
   - 参考: <https://mermaid.js.org/>

